api/v1alpha1/bgppeer_webhook.go

```diff
@@ -41,6 +42,20 @@ func (bgpPeer *BGPPeer) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 //+kubebuilder:webhook:verbs=create;update,path=/validate-metallb-io-v1alpha1-bgppeer,mutating=false,failurePolicy=fail,groups=metallb.io,resources=bgppeers,versions=v1alpha1,name=bgppeervalidationwebhook.metallb.io,sideEffects=None,admissionReviewVersions=v1
 
+var _ webhook.Defaulter = &BGPPeer{}
+
+// Default implements webhook.Defaulter so a webhook will be registered for the type
+func (bgpPeer *BGPPeer) Default() {
```

Comment:
Can we add an e2e that validates defaulting is working?

Reply:
ci isn't invoking the default settings seems like e2e issue need further investigation

Reply:
It has to do with some missing deployment / manifest setting I think.

Reply:
i dropped the use of default will address this separatly